pkg/resource/logging-secret/alloy-logging-secret.go

```diff
@@ -0,0 +1,106 @@
+package loggingsecret
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"text/template"
+
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/Masterminds/sprig/v3"
+
+	"github.com/giantswarm/logging-operator/pkg/common"
+	loggedcluster "github.com/giantswarm/logging-operator/pkg/logged-cluster"
+	loggingcredentials "github.com/giantswarm/logging-operator/pkg/resource/logging-credentials"
+)
+
+//go:embed alloy/logging.alloy.template
+var alloyLogging string
```

Comment:
Can we not group them using var (
  alloyLogging string
  alloyLoggingSecret string
)

Reply:
done